neqo-transport/src/connection/mod.rs

```diff
@@ -2562,6 +2614,7 @@ impl Connection {
                     RecoveryToken::RetireConnectionId(seqno) => self.paths.acked_retire_cid(*seqno),
                     RecoveryToken::AckFrequency(rate) => self.paths.acked_ack_frequency(rate),
                     RecoveryToken::KeepAlive => self.idle_timeout.ack_keep_alive(),
+                    RecoveryToken::Datagram => self.events.datagram_acked(),
```

Comment:
Do we really care about that?

Reply:
Probably not, I have added for now, just to have it when we start implemented the protocol on top of it.